deploy/resources/maistra/maistra-controlplane-0.10.0.yaml

```diff
@@ -20,6 +20,22 @@ spec:
         autoscaleEnabled: false
       istio-egressgateway:
         enabled: false
+      cluster-local-gateway:
+        autoscaleEnabled: false
+        enabled: true
+        labels:
+          app: cluster-local-gateway
+          istio: cluster-local-gateway
+          chart: gateways
+          heritage: Tiller
+          maistra-version: 0.10.0
+          release: istio
+        ports:
```

Comment:
Shall we specify the statusPort as well? See https://github.com/knative/serving/blob/master/third_party/istio-1.2.0/values.yaml#L43-L49

Reply:
Yes we can specify statusPort.
Added statusPort and updated PR.
Thank you